CONTRIBUTING.md

```diff
@@ -76,19 +76,19 @@ Check out the next section for details on how this plays out.
 
 [Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who have:
 
-1. Have actively engaged with Bevy development
+1. Have actively engaged with Bevy development.
 2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
 3. Have asked to join the Bevy Org. Reach out to @cart on Discord or email us at bevyengine@gmail.com if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
 
-Some Bevy Org members are also Triage Team members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Org members can vist the Bevy Engine Org page and navigate to the Triage Team to request membership. Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
+Some Bevy Org members are also Triage Team members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Org members can visit the Bevy Engine Org page and navigate to the Triage Team to request membership. Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
```

Comment:
All Bevy Org members are now able to triage issues and PRs. [Announced by cart here](https://discordapp.com/channels/691052431525675048/856713292067766382/981297942751240203). It would be nice to update this with that info.

Reply:
Agree!